tests/test_mantatail.py

```diff
@@ -123,6 +123,18 @@ def test_join_before_registering(run_server):
     assert received == b":mantatail 451 * :You have not registered\r\n"
 
 
+def test_join_channel(user_alice, user_bob):
+    user_alice.sendall(b"JOIN #foo\r\n")
+    user_bob.sendall(b"JOIN #foo\r\n")
+    received = receive_line(user_bob)
+    print(received)
+    assert received == b":Bob!Bob 0 * :Bob's real name@127.0.0.1 JOIN #foo\r\n"
```

Comment:
This isn't correct. IRC clients should be able to just assume that they can `.split(" ")`, but this won't work if you send the real name and it contains spaces.

To see what libera does, you can set a different real name in mantaray (right-click irc.libera.chat, "Server settings").

```suggestion
    assert received == b":Bob!BobUsr@127.0.0.1 JOIN #foo\r\n"
```